### PR TITLE
[zh-cn] update translation of {min,max}-width

### PR DESCRIPTION
**Pull request by jasonren0403**

### CODE REVIEW

File: files/zh-cn/web/css/max-width/index.md [hunk 1]
```diff
@@ -1,66 +1,79 @@
...
 title: max-width
 slug: Web/CSS/max-width
+l10n:
+  sourceCommit: 925b2bd8beeae6ce8237863637bcd28ccbb8d47f
 ---
 
 {{CSSRef}}
 
-## 摘要
+**`max-width`** [CSS](/zh-CN/docs/Web/CSS) 属性设置了元素的最大宽度。它可防止 {{cssxref("width")}} 属性的[应用值](/zh-CN/docs/Web/CSS/used_value)大于 `max-width` 指定的值。
```
Comment (yin1999) [inline files/zh-cn/web/css/max-width/index.md:10]:
```suggestion
**`max-width`** [CSS](/zh-CN/docs/Web/CSS) 属性设置元素的最大宽度。它可防止 {{cssxref("width")}} 属性的[应用值](/zh-CN/docs/Web/CSS/used_value)大于 `max-width` 指定的值。
```

File: files/zh-cn/web/css/max-width/index.md [hunk 2]
```diff
@@ -1,66 +1,79 @@
...
+- `fit-content`
+  - : 使用可用空间，但不得超过 [max-content](/zh-CN/docs/Web/CSS/max-content)，即 `min(max-content,max(min-content,stretch))`。
+- `fit-content({{cssxref("&lt;length-percentage&gt;")}})` {{Experimental_Inline}}
+  - : 使用 `fit-content` 公式，用指定参数替换可用空间，即 `min(max-content, max(min-content, argument))`。
+
+## 无障碍考虑
+
+确保设置了 `max-width` 的元素在页面缩放以增大文字大小时不会被截断和/或遮挡其他内容。
+
+- [MDN 理解 WCAG，指引 1.4 解释](/zh-CN/docs/Web/Accessibility/Understanding_WCAG/Perceivable#guideline_1.4_make_it_easier_for_users_to_see_and_hear_content_including_separating_foreground_from_background)
```
Comment (yin1999) [inline files/zh-cn/web/css/max-width/index.md:61]: Same as #18661. Could you align this

**Review by yin1999 — APPROVED** [review]
Looks good. Just some nits:

File: files/zh-cn/web/css/max-width/index.md (hunk 2 above)
Comment (jasonren0403) [reply files/zh-cn/web/css/max-width/index.md:61]:
```suggestion
- [MDN 理解 WCAG 准则 1.4](/zh-CN/docs/Web/Accessibility/Understanding_WCAG/Perceivable#guideline_1.4_make_it_easier_for_users_to_see_and_hear_content_including_separating_foreground_from_background)
```

**Pull request merged**